### DISCUSSION TITLE
macOS support

### DISCUSSION THREAD

**Discussion by ClarusAD** [localsend/localsend#1167] [Discussion (English, International)]
Hello,
Is there a macOS 10.14 (Mojave) version ?
Thank-you

**Comment by changenowde** [comment]
I would love such a version too - especially for my 2011 Macbook Pro that is still in use.

**Comment by dajare** [comment]
I don't know whether "+1s" ("Me too!") are welcome in this king of thread but ... I have a 2015 MacBook still running Mojave (MUST have 32-bit capability for a particular application I depend on). It would be great to be able to have LocalSend on this machine, too.

**Comment by chuchana** [comment]
Have you tried [LocalSend Web](https://web.localsend.org/)?
